tests/unit/src/kakarot/accounts/eoa/test_library.py

```diff
@@ -32,9 +37,10 @@ class TestLibrary:
     async def test_execute_should_make_all_calls_and_return_concat_results(
         self, mock_externally_owned_account, eth
     ):
-        private_key = generate_random_private_key()
+        private_key = keys.PrivateKey(int.to_bytes(0xDEAD, 32, "big"))
```

Comment:
is 0xDEAD being used here because of a convention of an account deployed on devnet? 

maybe it can be defined as a constant where the variable name offers some clarity.

Reply:
And can't remember who introduced this convention in the project to use this 0XDEAD as a mean of "a random number we don't care about".

Could move this to a DEFAULT_PRIVATE_KEY in a constant file if you prefer, not sure it's much better though

Reply:
ah, wasn't familiar with this!

if the `0xDEAD` usage is broader than just a default private key and just used as a random filler value, then maybe there's a place in testing/deployment docs where that can be mentioned

Reply:
doing search for all "0xDEAD", it's used as a magic number for addresses & private keys in tests. I've removed all the usage apart from one explicit one `test__exec_extcodehash__should_handle_invalid_address` where `0xDEAD` is clearly used to be "the invalid address"